src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -11,7 +11,7 @@
 from alchemtest.amber import load_invalidfiles
 from alchemtest.amber import load_bace_example
 from alchemtest.amber import load_bace_improper
-
+from numpy import isclose 
```

Comment:
Standard order of imports (standard lib, framework packages like numpy, specific packages like alchemtest, own modules) would have numpy imports before alchemtest... although I am not sure if we're consistent.

Reply:
Thanks, I fixed the order!